test/prerna/testing/reactor/export/ToXmlReactorTests.java

```diff
@@ -0,0 +1,333 @@
+package prerna.testing.reactor.export;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.jupiter.api.Test;
+
+import prerna.algorithm.api.SemossDataType;
+import prerna.reactor.export.IterateReactor;
+import prerna.reactor.export.ToXmlReactor;
+import prerna.reactor.qs.source.DatabaseReactor;
+import prerna.sablecc2.om.ReactorKeysEnum;
+import prerna.sablecc2.om.nounmeta.NounMetadata;
+import prerna.testing.AbstractBaseSemossApiTests;
+import prerna.testing.ApiSemossTestEngineUtils;
+import prerna.testing.ApiSemossTestInsightUtils;
+import prerna.testing.ApiSemossTestUtils;
+import prerna.testing.PixelChain;
+
```

Comment:
Testing Multiple rows, One row, Every data type supported (except factor). I believe Factor is somethign with R. Going to work on getting R tests running this week. Also added boolean support for table upload so I can export a boolean value here so I didn't have to write a new way to generate dbs.